AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,9 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+bo.de#$##footer-consent { display: none !important; }
+bo.de#$##consent-overlay { display: none !important; }
+bo.de#$#body { overflow: visible !important; padding-right: 0 !important; }
```

Comment:
Not seeing a difference with padding-right.

Reply:
Accidentally added.